packages/react-client/src/useSetupMedia.ts

```diff
@@ -51,7 +46,7 @@ export const createUseSetupMediaHook = <PeerMetadata, TrackMetadata>(
             pending = true;
 
             await videoTrackManager
-              .startStreaming(config.defaultTrackMetadata, config.defaultSimulcastConfig, config.defaultMaxBandwidth)
+              .startStreaming(config.defaultSimulcastConfig, config.defaultMaxBandwidth)
               .finally(() => {
                 pending = false;
               });
```

Comment:
I don't think we should mix `thenable` syntax with `awaitable` 🤔 
```suggestion
            try {
              await videoTrackManager.startStreaming(config.defaultSimulcastConfig, config.defaultMaxBandwidth);
            } finally {
              pending = false;
            }
```
(this applies to other places in this file too - so maybe better to fix it in separate PR)